### ISSUE TITLE
Don't write trace summary report when exporting VFB data

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-bwtrace#100]
Reported by @andrewmleonard "I get this "stuff" if I export VFB data" following by the trace summary report.
VFB is Visual Form Builder 

The URL is of the form `https://s.b/wordpress/wp-admin/admin.php?page=vfb-export`

The $_REQUEST paramters are:
_REQUEST Array
```
    [page] => (string) "vfb-export"
    [vfb-content] => (string) "entries"
    [format] => (string) "csv"
    [entries_form_id] => (string) "1"
    [entries_start_date] => (string) "0"
    [entries_end_date] => (string) "0"
    [submit] => (string) "Download Export File"
```

**Comment by bobbingwide** [comment]
Solution. return false from  `bw_trace_ok_to_echo()` when there's a `vfb-content` query parm.

**Comment by bobbingwide** [comment]
Released in v3.2.2. Closing therefore
